tests/unit/algos/list/delimiter.js

```diff
@@ -273,6 +273,181 @@ const tests = [
         IsTruncated: false,
         NextMarker: undefined,
     }, (e, input) => e.key > input.marker),
+
+    new Test('all elements v2', {
+        v2: true,
+    }, {
+        Contents: receivedData,
+        CommonPrefixes: [],
+        Delimiter: undefined,
+        IsTruncated: false,
+        NextContinuationToken: undefined,
+    }),
+    new Test('with valid startAfter', {
+        startAfter: receivedData[4].key,
+        v2: true,
+    }, {
+        Contents: [
+            receivedData[5],
+            receivedData[6],
+            receivedData[7],
+            receivedData[8],
+            receivedData[9],
+        ],
+        CommonPrefixes: [],
+        Delimiter: undefined,
+        IsTruncated: false,
+        NextContinuationToken: undefined,
```

Comment:
Thinking out loud here: are we interested in returning the `StartAfter` value, or is that just handled by Cloudserver in the XML response?

Reply:
Yeah, it's handled by Cloudserver